src/app/devices/kvm/kvm.component.ts

```diff
@@ -97,6 +97,12 @@ export class KvmComponent implements OnInit, AfterViewInit, OnDestroy {
     this.module.onSend = this.redirector.send.bind(this.redirector)
     this.module.onProcessData = this.dataProcessor.processData.bind(this.dataProcessor)
     this.module.bpp = this.selected
+    this.mouseMove = fromEvent(this.canvas?.nativeElement, 'mousemove')
+    this.mouseMove.pipe(throttleTime(200)).subscribe((event: any) => {
```

Comment:
are you sure you want throttleTime over debounce? I think you'd want debounce. The main difference is that the last value will be emitted once the time elapses.